static/app/utils/replays/hydrateBreadcrumbs.tsx

```diff
@@ -2,13 +2,26 @@ import invariant from 'invariant';
 
 import {BreadcrumbType} from 'sentry/types/breadcrumbs';
 import isValidDate from 'sentry/utils/date/isValidDate';
-import type {BreadcrumbFrame, RawBreadcrumbFrame} from 'sentry/utils/replays/types';
-import {isBreadcrumbFrame} from 'sentry/utils/replays/types';
+import type {
+  BreadcrumbFrame,
+  RawBreadcrumbFrame,
+  RecordingFrame,
+} from 'sentry/utils/replays/types';
+import {EventType, isBreadcrumbFrame} from 'sentry/utils/replays/types';
 import type {ReplayRecord} from 'sentry/views/replays/types';
 
+function findNextMutation(date: Date, rrwebFrames: RecordingFrame[]): RecordingFrame {
+  const timeMS = date.getTime();
+  const framesAfter = rrwebFrames.filter(
+    frame => frame.type === EventType.IncrementalSnapshot && frame.timestamp > timeMS
+  );
+  return framesAfter[0] ?? {};
+}
```

Comment:
Since this is a POC, let's find the nearest mutation before *and* after so we can verify how consistent this data is?

Reply:
Ack

Reply:
done!